modules/core/arrow-annotations-processor/src/main/java/arrow/optics/BoundSetterGenerator.kt

```diff
@@ -41,16 +41,16 @@ class BoundSetterGenerator(
                |import $packageName.*
                |""".trimMargin()
 
-  fun createDslFunction(annotatedOptic: AnnotatedOptic): String = """
+  private fun createDslFunction(annotatedOptic: AnnotatedOptic): String = """
         |/**
         | * @receiver [${annotatedOptic.sourceClassName.removeBackticks()}] the instance you want to bind the dsl on.
         | * @return [$boundSetter] an intermediate optics that is bound to the instance.
         | */
         |fun ${annotatedOptic.sourceClassName}.setter() = $boundSetter(this, arrow.optics.PSetter.id())
         |""".trimMargin()
 
-  fun processBoundSetter(sourceClassName: String, targetName: String, targetClassName: String, sourceName: String) = """
-      |inline val <T> $boundSetter<T, $sourceClassName>.$targetName: $boundSetter<T, $targetClassName>
+  private fun processBoundSetter(sourceClassName: String, targetName: String, targetClassName: String, sourceName: String) = """
```

Comment:
maybe reorder to `sourceName, sourceClassName, targetName, targetClassName` ? The args feel quite disordered now